### PR TITLE
Deliverable C: Review for Documentation and Comments. GROUP D

### PR DESCRIPTION
**Pull request by manoj-goli**
Hi,

Could you please review the documentation and code comments as part of deliverable C

Files assigned to Manoj are:
- main.cpp (test/src/sender),
- sendercadmium.hpp (include/atomics).

Files assigned to Revanth are:
- receivercadmium.hpp (include/atomics),
- subnetcadmium.hpp (include/atomics).

README file has been changed as per markdown notations.

Thanks!

### CODE REVIEW

**Comment by shubhamthakur463** [comment]
In the include/atomics folder, please change the name of all 3 header files according to the coding conventions.

Sorry for not pointing it out during PART-B(Coding Conventions) review.

**Comment by manoj-goli** [comment]
> In the include/atomics folder, please change the name of all 3 header files according to the coding conventions.
> 
> Sorry for not pointing it out during PART-B(Coding Conventions) review.

Hi,
Updated.
Thanks.

File: test/src/sender/main.cpp [hunk 1]
```diff
@@ -16,51 +25,86 @@
...
 
-    /*************** Loggers *******************/
+    /**
+     * Generating messages and operation logs which are being fed
+     * at execution time of this application and storing them 
+     * in the respective path as mentioned to sender_test_output 
+     * file as indicated in the out_data.
+     */
     static std::ofstream out_data(
         "../test/data/sender/sender_test_output.txt"
```
Comment (gagandeep8030) [inline test/src/sender/main.cpp]: File path here is hardcoded which need to be softcoded using #define. Kindly make the changes in all the .cpp files (top_model, receiver, sender, subnet).

Comment (manoj-goli) [reply test/src/sender/main.cpp]: Hi,

Updated, thanks. Please review and approve the request.

Thanks

**Review by revanth-sridhar — APPROVED** [review]
Project documentation (including README file) and code comments pull request is in sync with the requirements established for the project as part of deliverable C.

**Review by shubhamthakur463 — APPROVED** [review]
Approved.

**Review by gagandeep8030 — APPROVED** [review]
Approved Part C

**Pull request merged**